swap_meet/vendor.py

```diff
@@ -1,2 +1,106 @@
 class Vendor:
-    pass
\ No newline at end of file
+    ''' 
+    Represents a Vendor with an invetory of items. 
+    Attributes: inventory with list of items.
+    Methods: add(item): Adds item to inventory, and returns item.
+            remove(item): Removes and returns item, or False if not found.
+    '''
+    
+    def __init__(self, inventory = None):
+        self.inventory = [] if inventory is None else inventory
+  
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
+        else:
+            self.inventory.remove(item)
+            return item
+
+    def get_by_id(self, vendor_id):
```

Comment:
I would maybe change this to `item_id` since we are looking for a specific item by id not necessarily a vendor.